unix/xserver/hw/vnc/vncModule.c

```diff
@@ -36,7 +36,7 @@
 #include "XorgGlue.h"
 #include "RandrGlue.h"
 
-static void vncModuleInit(INITARGS);
+static void vncModuleInit(void);
```

Comment:
Good catch. This looks like it should have been included in e4ec992984983ac9107efae1f1cc552007e4229e.